modules/statics/src/base.ts

```diff
@@ -143,6 +143,7 @@ export const enum UnderlyingAsset {
   ZEC = 'zec',
 
   // ERC 20 tokens
+  '1UP' = '1up',
```

Comment:
Keys can't start with numbers apparently, so had to wrap in quotes

Reply:
Yep, they follow the normal identifier rules